### ISSUE TITLE
Shaperglot is crashing

### ISSUE THREAD

**Issue by Hoolean** [googlefonts/googlesans-flex#1045]
Shaperglot's latest version is crashing - @simoncozens I've reported upstream at googlefonts/shaperglot#81 but shooting you a ping as here I can share all of the juicy details :)

- [CI run](https://github.com/googlefonts/googlesans-flex/actions/runs/10901330135/job/30251112626#step:12:102)
- Full traceback:
```
Traceback (most recent call last):
  File "/home/runner/work/googlesans-flex/googlesans-flex/venv/bin/shaperglot", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/runner/work/googlesans-flex/googlesans-flex/venv/lib/python3.11/site-packages/shaperglot/cli/__init__.py", line 74, in main
    options.func(options)
  File "/home/runner/work/googlesans-flex/googlesans-flex/venv/lib/python3.11/site-packages/shaperglot/cli/report.py", line 28, in report
    if options.glyphset:
       ^^^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'glyphset'
```

**Comment by simoncozens** [comment]
Bah. Can be worked around by installing gfglyphsets, but I'll fix it.

**Comment by simoncozens** [comment]
fixed in 0.6.3

**Comment by Hoolean** [comment]
Fix confirmed in the original CI run, v0.6.3 now working great; cheers Simon 🚀
